app/templates/components/public/side-menu.hbs

```diff
@@ -3,32 +3,44 @@
     {{#scroll-to href='#info' class='item active'}}
       {{t 'Info'}}
     {{/scroll-to}}
-    {{#scroll-to href='#tickets' class='item'}}
-      {{t 'Tickets'}}
-    {{/scroll-to}}
-    {{#scroll-to href='#speakers' class='item'}}
-      {{t 'Speakers'}}
-    {{/scroll-to}}
+    {{#if event.tickets}}
+      {{#scroll-to href='#tickets' class='item'}}
+        {{t 'Tickets'}}
+      {{/scroll-to}}
+    {{/if}}
+    {{#if event.speakers}}
+      {{#scroll-to href='#speakers' class='item'}}
+        {{t 'Speakers'}}
+      {{/scroll-to}}
+    {{/if}}
   {{else}}
-    <a class="item" href="{{href-to 'public.index'}}">
-      {{t 'Info'}}
-    </a>
+  <a class="item" href="{{href-to 'public.index'}}">
+    {{t 'Info'}}
+  </a>
+  {{#if event.isTicketingEnabled}}
     <a class="item" href="{{href-to 'public.index'}}#tickets">
       {{t 'Tickets'}}
     </a>
+  {{/if}}
+  {{#if event.speakers}}
```

Comment:
This will cause a network request to speakers. Use `is-sessions-speakers-enabled ` from events response for this.